tests/app_test.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "../lib/passageidentity/client"
+require "dotenv"
+require "faraday"
+require "test/unit"
+
+Dotenv.load(".env")
+class TestAppAPI < Test::Unit::TestCase
+  PassageClient =
+    Passage::Client.new(app_id: ENV["APP_ID"], api_key: ENV["API_KEY"])
+
+  def test_get_app()
+    app = PassageClient.get_app()
+    assert_equal ENV["APP_ID"], app.id
+  end
```

Comment:
Do you think we should include a fail case here to check if we're handling the error properly?